src/Http/Routing/src/Matching/LinearSearchJumpTable.cs

```diff
@@ -30,17 +30,11 @@ public override int GetDestination(string path, PathSegment segment)
         }
 
         var entries = _entries;
+        var pathSpan = path.AsSpan(segment.Start..(segment.Start + segment.Length));
```

Comment:
```suggestion
        var pathSpan = path.AsSpan(segment.Start, segment.Length);
```
* simpler to read
* less work for compilers

Reply:
Thank you for the suggestion!